server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -108,6 +110,20 @@ public void testCopyFromFileWithJsonExtension() throws Exception {
         assertThat(response.rows()[0][0], is("Don't pa\u00f1ic."));
     }
 
+    @Test
+    public void testCopyFromFileNoWaitForCompletion() throws Exception {
+        execute("create table quotes (id int primary key, " +
+                "quote string index using fulltext) with (number_of_replicas = 0)");
+
+        execute("copy quotes from ? with (format='csv', wait_for_completion=false)",
+                new Object[]{copyFilePath + "test_copy_from_csv.ext"});
+        assertThat(response.rowCount(), either(is(0L)).or(is(-1L)));
```

Comment:
How can this be `0`? 
Anyhow, I think we can create a unit test instead, see my previous comment.

Reply:
Most of the times is 0! But I ll remove it completely!

Reply:
If it's `0` there must be a bug as your code always return a `-1` row.

Reply:
Yes that was odd. Most times is 0 but some times is -1. How can this happen?

Reply:
I removed the integration test but the odd behaviour persisted. Some times was -1 some times 0.

Reply:
Is this issue resolved? In the latest revision `CopyIntegrationTest` changes are removed (except imports) - I wonder what happens if we run on the current revision? If this test is not needed (??) then please revert resudual changes in `CopyIntegrationTest`

Reply:
@BaurzhanSakhariev The issue existed as long as the test existed. @seut suggested above to create a unit test instead so i removed it. Unit test works fine and never had an issue with manual tests. 
I will remove the unused imports :)

Reply:
>never had an issue with manual tests.

if it's always -1 with manual testing it might be wrong test setup (then it's ok to have only unit tests if it's backed by real manual testing). But I'm really wondering what could be wrong in such small test case - I hope it doesn't indicate than it highlights some bug which is somehow slipped away in manual testing.